include/frozen/bits/algorithms.h

```diff
@@ -133,10 +139,10 @@ constexpr void quicksort(Iterator left, Iterator right, Compare const &compare)
   }
 }
 
-template <typename T, std::size_t N, class Compare>
-constexpr bits::carray<T, N> quicksort(bits::carray<T, N> const &array,
+template <typename Container, class Compare>
```

Comment:
Can you motivate that particular change?

Reply:
This is preparation for adding a new container type specialized for storing strings more efficiently.

My current WIP commit for that is muggenhor/frozen@0714d53cc9a4c030faaaed6a4b5e7503458293ff (part of the [feat/zero-reloc-string-containers](https://github.com/muggenhor/frozen/compare/fixes...muggenhor:frozen:feat/zero-reloc-string-containers) branch I linked above).